### ISSUE TITLE
Latest version (0.12.0) broken

### ISSUE THREAD

**Issue by yhnavein** [petyosi/react-virtuoso#43]
When upgraded to the latest version (0.12.0), my app is not buildable by TypeScript because of the following error:

```
TypeScript error in app/node_modules/react-virtuoso/dist/Virtuoso.d.ts(2,29):
Cannot find module 'tinyrx'.  TS2307

    1 | import React, { CSSProperties, FC, PureComponent, ReactElement } from 'react';
  > 2 | import { TSubscriber } from 'tinyrx';
      |                             ^
    3 | import { TScrollLocation } from './EngineCommons';
    4 | import { ListItem } from './GroupIndexTransposer';
    5 | import { TRender, TRenderProps } from './VirtuosoList';
```

I can see that `react-virtuoso` does not refer to the `tinyrx` as its dependency.

**Comment by yhnavein** [comment]
And it seems that the same error appears in the version `0.11.1`
Version `0.10.2` works fine

**Comment by petyosi** [comment]
👍 Thanks for reporting this, fixed the import in 0.12.1, let me know if it works for you. 

Cheers,

**Comment by EdwardSalter** [comment]
I am seeing this exact issue with the latest version (0.17.4) also whilst using TypeScript.
```
TypeScript error in C:/dev/,myapp/node_modules/react-virtuoso/dist/VirtuosoGrid.d.ts(2,29):
Cannot find module 'tinyrx' or its corresponding type declarations.  TS2307

    1 | import React, { CSSProperties, ReactElement } from 'react';
  > 2 | import { TSubscriber } from 'tinyrx';
```

**Comment by petyosi** [comment]
@EdwardSalter - sorry about that, should be fixed in `0.17.5`.

**Comment by EdwardSalter** [comment]
@petyosi - Thank you for that.
Do you have an estimate on when that will be released?
